Week1/homework.md

```diff
@@ -0,0 +1,29 @@
+# Homework QA Training Week 1: Foundation to Software testing
```

Comment:
Awesome work Mustafa 👏 ! I like the clear differentiation of unique activities per platform 👍. I am curious about few things so I just added some inline comments. please feel free to comment back with your point of view :)